docker-compose.test.yml

```diff
@@ -0,0 +1,24 @@
+version: "3.7"
+services:
+  test:
```

Comment:
Having this around for testing purposes is something I would have found useful when debugging CI so thank you 👍 

Usually for this type of work I need to give docker compose access to my local files so changes I make are made available without having to rebuild the image everytime. [I'd usually do something like this](https://github.com/LBHackney-IT/report-a-defect/blob/develop/docker-compose.test.yml#L15) in a combination with running the container with Spring to autoload the changes.

I think this change could be merged in without this suggestion as the only downside is that rebuilding takes more time (if indeed this is an issue).

Reply:
Ah! Good shout, I will change that.

Reply:
Spring is something I have little understanding of, it would be good to use this to learn more. I'll definately add the volumes though.